app/views/posts/show.html.erb

```diff
@@ -1,2 +1,25 @@
 <h1>Posts#show</h1>
```

Comment:
- [ ] I would like to recommend you remove this `<h1> ` element and make sure and Post show page is displayed 1. on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) as required for this project.  for example, the user photo is square, not rounded. Please fix it :+1:

Reply:
Thanks for the review @codecaiine although I get the issues you've raised I would much appreciate it if the review was more based on the implementation of the functionalities rather than styling. so far as wireframes go the fact that the photo is a square in the wireframe doesn't mean it has to be since that has nothing to do with functionality.

![wireframe](https://user-images.githubusercontent.com/89058561/208217941-0d2fe2d5-82a3-4d55-8e6f-f3e5d5c4897c.png)


Thank You!